### ISSUE TITLE
Vscode Fix-all auto fixable problem not working

### ISSUE THREAD

**Issue by sgywzy** [microsoft/vscode-eslint#832]
Vscode(latest): 1.41.0
Vscode extension: Eslint 2.0.4

```
  "devDependencies": {
    "eslint": "^6.7.2",
    "eslint-config-airbnb-base": "^14.0.0",
    "eslint-plugin-import": "^2.19.1"
  }
```

Auto-update is turned on by default and i got the latest version too (2.0.4) which broke the 
Fix-all auto-fixable Problems function. Every problem shows up correctly but it's unable to correct it.
It's on a hotkey for me but invoking directly from the console(F1->...) doesn't do anything either.

As a quick fix, you can navigate to the extension tab, disable auto-updates then downgrade to a previous build.

**Comment by dracozombie19** [comment]
I am experiencing the same issue.  VSCode just updated and the extension along with it.  I've been monkeying around with the new settings that the extension provides and I've been able to get it to apply auto-fixes using the Format Document action, but explicitly running the auto-fix command or trying to setup auto-fix on save is not working.

I'm still running ESLint version 5.16.0, for reference.

**Comment by dbaeumer** [comment]
@sgywzy @dracozombie19 my mistake and thanks for reporting. Reason is that I moved the fix all to the source action menu. You can now do 

> Context Menu > Source Action which should give you:

<img width="479" alt="capture" src="https://user-images.githubusercontent.com/1931590/70940028-c3543a80-2049-11ea-912b-cc6d2ba7e332.png">

Executing the action will fix all ESLint errors.

You can also use `Fix All` from the command palette which works as well. I will think about what to do with the ESLint specific command. My current thinking is to show a notification that the command got integrated into the source actions. Preferences ?

**Comment by dbaeumer** [comment]
I rebound the command to the new source action code.

**Comment by dbaeumer** [comment]
Published 2.0.5
